### ISSUE TITLE
How can I modify the environment variables which built from other env variables?

### ISSUE THREAD

**Issue by umaYnit** [sagiegurari/cargo-make#520]
This is My makefile.toml: 
```toml
[config]
skip_core_tasks = true

[env]
MODE = { value = "release", condition = { env_not_set = ["MODE"] } }
KERNEL_ELF = "target/${MODE}/kernel.bin"

[tasks.print]
script_runner = "@duckscript"
script = '''
echo ${MODE}
echo ${KERNEL_ELF}
'''

[tasks.debug]
command = "cargo"
args = ["make", "print", "--env", "MODE=debug"]
```

**Comment by sagiegurari** [comment]
not sure what is the question here, your makefile seems to work well:

```console
~/ $ cargo make print
[cargo-make] INFO - cargo make 0.32.12
[cargo-make] INFO - Build File: Makefile.toml
[cargo-make] INFO - Task: print
[cargo-make] INFO - Profile: development
[cargo-make] INFO - Running Task: print
release 
target/release/kernel.bin 
[cargo-make] INFO - Build Done in 0.00 seconds.
~/ $ cargo make debug
[cargo-make] INFO - cargo make 0.32.12
[cargo-make] INFO - Build File: Makefile.toml
[cargo-make] INFO - Task: debug
[cargo-make] INFO - Profile: development
[cargo-make] INFO - Execute Command: "cargo" "make" "print" "--env" "MODE=debug"
[cargo-make][1] INFO - Build File: Makefile.toml
[cargo-make][1] INFO - Task: print
[cargo-make][1] INFO - Profile: development
[cargo-make][1] INFO - Running Task: print
debug 
target/release/kernel.bin 
[cargo-make][1] INFO - Build Done in 0.00 seconds.
[cargo-make] INFO - Build Done in 0.56 seconds.
```

**Comment by umaYnit** [comment]
I expect that when the variable `MODE` is debug, the variable `KERNEL_ELF` can be `"target/debug/kernel.bin"`

**Comment by sagiegurari** [comment]
@umaYnit this should now be resolved in dev branch 0.32.13

**Comment by umaYnit** [comment]
Yes, it's ok, I can test code conveniently and happily again. thanks~ :laughing:
